service/controller/v25/key/key.go

```diff
@@ -47,6 +47,9 @@ const (
 	// LogDeliveryURI is used for setting the correct ACL in the access log bucket
 	LogDeliveryURI = "uri=http://acs.amazonaws.com/groups/s3/LogDelivery"
 
+	// S3BucketEncryptionAlgorithm is used to determine which algorithm use S3 to encrypt buckets.
+	S3BucketEncryptionAlgorithm = "AES256"
```

Comment:
The `key` package is overloaded with all kinds of information not relevant for multiple packages. Let's keep the keys and constants relevant for a single package in the package that is interested in the information. That way the key package should become leaner and only hold information relevant in multiple places.

Reply:
ok so I create a new key package under the resource? or do I include the key in same file (resource/create.go)?

Reply:
I think it is fine to just put it into the file where it is used.